src/seccomp.c

```diff
@@ -81,17 +84,59 @@ int pg_init_seccomp(void)
 		ALLOW_SYSCALL(gettimeofday),
 		ALLOW_SYSCALL(stat),
 		ALLOW_SYSCALL(clock_gettime),
+		ALLOW_SYSCALL(mprotect),
```

Comment:
humm, This one is not acceptable.
I understand you need it to debug, but this call is far too dangerous.